charts/minibroker/templates/rbac.yaml

```diff
@@ -20,8 +57,147 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cluster-admin
+  name: minibroker
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.rbac.serviceAccount.name | quote }}
+  namespace: {{ .Release.Namespace | quote }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: minibroker
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["*"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: minibroker
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: minibroker
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.rbac.serviceAccount.name | quote }}
+  namespace: {{ .Release.Namespace | quote }}
+{{- range $namespace := .Values.rbac.namespaced.whitelist }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: minibroker
+  namespace: {{ $namespace | quote }}
+  labels:
+    app: {{ template "fullname" $ }}
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
+    release: "{{ $.Release.Name }}"
+    heritage: "{{ $.Release.Service }}"
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: minibroker
+  namespace: {{ $namespace | quote }}
+  labels:
+    app: {{ template "fullname" $ }}
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
+    release: "{{ $.Release.Name }}"
+    heritage: "{{ $.Release.Service }}"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: minibroker
 subjects:
 - kind: ServiceAccount
+  name: {{ $.Values.rbac.serviceAccount.name | quote }}
+  namespace: {{ $.Release.Namespace | quote }}
+{{- end }}{{/* range $namespace := .Values.rbac.namespaced.whitelist */}}
+{{- else }}{{/* if .Values.rbac.namespaced.enabled */}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: minibroker
```

Comment:
Do we need to worry about having multiple minibrokers deployed into one cluster (possibly across different kubecf / svcat instances)? (i.e. should `.Release.Name` be involved in some way?)

Reply:
Good catch. We do need to get `.Release.Name` into the cluster role name.

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/65/commits/6f7675e2984304cd239bbe73018430cf371a5237.